auxiliary/auxiliary_test.go

```diff
@@ -43,20 +43,56 @@ func TestParseArgs(t *testing.T) {
 	}
 
 	args = []string{"./pocket", "help", "test"}
-	mode, err = ParseArgs(args)
-	if err.Error() != "[!] Invalid arguments" {
+	mode, _, err = ParseArgs(args)
+	if err.Error() != "[!] Invalid mode passed to help" {
 		t.Error("Expected error; got ", err)
 	}
 	if mode != "" {
 		t.Error("Expected empty mode; got", mode)
 	}
 
 	args = []string{"./pocket", "test"}
-	mode, err = ParseArgs(args)
-	if err.Error() != "[!] Invalid arguments" {
+	mode, _, err = ParseArgs(args)
+	if err.Error() != "[!] Invalid option" {
 		t.Error("Expected error; got ", err)
 	}
 	if mode != "" {
 		t.Error("Expected empty mode; got", mode)
 	}
+
+	args = []string{"./pocket", "get", "-c"}
```

Comment:
It's being passed to a function that takes a slice of arguments, so it's not actually running anything, just emulating it being called with certain arguments.